### ISSUE TITLE
Pull the sitemap tests into a separate scheduled job

### ISSUE THREAD

**Issue by U-DON** [department-of-veterans-affairs/va.gov-team#21476]
## Background

Currently, the sitemap tests (AKA `nightwatch-accessibility`) can take nearly 40 minutes to run.
Each VAMC rollout adds around 200 pages to the sitemap and 3-4 minutes to the test suite.

There is a separate effort to implement a unit testing framework for the Liquid templates to test accessibility in any changes to those templates. In combination with aXe checks that are required in Cypress tests for VFS teams by Product Support, we can ensure accessibility tests for the changes that developers make within `vets-website` code.

CMS content itself has the potential to introduce accessibility errors. But as the content is not tied to changes that `vets-website` developers introduce themselves, the sitemap tests should not block those changes from going out. We could rely on the sitemap tests running asynchronously for this coverage. It should also be noted that accessibility errors introduced by content editors are fairly rare.

Furthermore, having this separate job would enable accessbility specialists (or anyone else) to run the tests and generate a report of issues on demand.

Goals:
* Limit the impact of the growing sitemap on the build time
  - The sitemap tests are currently run in parallel with E2E tests, which themselves take ~30 minutes.
  - Removing them from the pipeline would then make the E2E tests the bottleneck, saving us ~10 minutes.
* Maintain accessibility test coverage across all pages on the site

## Acceptance criteria

- [ ] The `nightwatch-accessbility` step is removed from the Jenkins pipeline
- [ ] A new job is created to run the equivalent of `nightwatch-accessibility` daily at 4:30pm ET

**Comment by meganhkelley** [comment]
Meeting notes from 3/12: https://docs.google.com/document/u/1/d/1oMOHVDjM5OCLl3DMSQJvCJ4X6_5anHs4ysv4w5-0rc8/edit#heading=h.3y22dzax96ny

**Comment by timwright12** [comment]
Devops PR: https://github.com/department-of-veterans-affairs/devops/pull/8712

vets-website PR: https://github.com/department-of-veterans-affairs/vets-website/pull/16375

**Comment by meganhkelley** [comment]
Relevant Slack thread: https://dsva.slack.com/archives/CQH357ZTP/p1615909757002800

Dror Matalon  2 hours ago
So I've been instructed that we can remove the site-map accessibility checks from the branch commits but we have to keep them when merging to master.
So I guess we don't then need to implement the new daily checks.
The times for these will increase as we add new facilities.
Good news: when we do the build content separation, this will stop being an issue for the daily build.
Bad news: I guess we should be running these tests as part of the content build which'll make the one hour build times tight.
The long term solution for this, like everything else related to the content build is that we should just run the tests of pages that have changed.
I wonder if there's a way to run these tests inside Drupal before publication.
